Bruin Bite/View Controllers/MenuVC.swift

```diff
@@ -38,6 +38,17 @@ class MenuVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSo
     let diningHalls = ["De Neve", "BPlate", "Covel", "Feast"]
     var menuItem: Item?
     
+    var loading_1: UIImage!
+    var loading_2: UIImage!
+    var loading_3: UIImage!
+    var loading_4: UIImage!
+    var loading_5: UIImage!
+    var loading_6: UIImage!
+    var images: [UIImage]!
```

Comment:
You shouldn't need to make six variables to build an array. Just use an array and a for loop to build the array of images.